iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/integration/test/IcebergRESTADLSIT.java

```diff
@@ -0,0 +1,128 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.iceberg.integration.test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.gravitino.catalog.lakehouse.iceberg.IcebergConstants;
+import org.apache.gravitino.credential.CredentialConstants;
+import org.apache.gravitino.iceberg.common.IcebergConfig;
+import org.apache.gravitino.integration.test.util.BaseIT;
+import org.apache.gravitino.integration.test.util.DownloaderUtils;
+import org.apache.gravitino.integration.test.util.ITUtils;
+import org.apache.gravitino.storage.ADLSProperties;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+@SuppressWarnings("FormatStringAnnotation")
+@EnabledIfEnvironmentVariable(named = "GRAVITINO_TEST_CLOUD_IT", matches = "true")
+public class IcebergRESTADLSIT extends IcebergRESTJdbcCatalogIT {
+
+  private String storageAccountName;
+  private String storageAccountKey;
+  private String tenantId;
+  private String clientId;
+  private String clientSecret;
+  private String warehousePath;
+
+  @Override
+  void initEnv() {
+    this.storageAccountName =
+        getFromEnvOrDefault("GRAVITINO_ADLS_STORAGE_ACCOUNT_NAME", "{STORAGE_ACCOUNT_NAME}");
+    this.storageAccountKey =
+        getFromEnvOrDefault("GRAVITINO_ADLS_STORAGE_ACCOUNT_KEY", "{STORAGE_ACCOUNT_KEY}");
+    this.tenantId = getFromEnvOrDefault("GRAVITINO_ADLS_TENANT_ID", "{TENANT_ID}");
+    this.clientId = getFromEnvOrDefault("GRAVITINO_ADLS_CLIENT_ID", "{CLIENT_ID}");
+    this.clientSecret = getFromEnvOrDefault("GRAVITINO_ADLS_CLIENT_SECRET", "{CLIENT_SECRET}");
+    this.warehousePath =
+        String.format(
+            "abfss://%s@%s.dfs.core.windows.net/data/test",
+            getFromEnvOrDefault("GRAVITINO_ADLS_CONTAINER", "{ADLS_CONTAINER}"),
+            storageAccountName);
+
+    if (ITUtils.isEmbedded()) {
```

Comment:
Could the IT run pass in deploy mode?   Please refer to https://gravitino.apache.org/docs/0.7.0-incubating/how-to-test for more details about how to test on deploy mode.

Reply:
I tested it, and it passed in deploy mode.